packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -210,6 +245,31 @@ export class ProjectDataSource {
     this._specWatcher.on('unlink', onSpecsChanged)
   }
 
+  async defaultSpecFileName () {
+    const defaultFileName = 'cypress/e2e/filename.cy.js'
```

Comment:
Is this fallback needed? Won't our default specPattern be parsed to determine this value from `getDefaultSpecFileName`?

Reply:
It's being used if we are not able to generate the fileName, or there's an unexpected error!